apps/src/templates/teacherDashboard/RosterDialog.jsx

```diff
@@ -144,7 +151,10 @@ export default class RosterDialog extends React.Component {
         {...this.props}
       >
         <h2 style={styles.title}>
-          {locale.selectGoogleClassroom()}
+          {this.props.provider === 'google' ?
+            locale.selectGoogleClassroom() :
+            locale.selectCleverClassroom()
+          }
```

Comment:
Are we likely to have more providers soon? If so, maybe this should be a switch instead of ternary.